package.json

```diff
@@ -17,31 +17,31 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "next": "10.2.0",
-    "next-seo": "4.24.0",
+    "next": "11.0.1",
+    "next-seo": "4.26.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-is": "17.0.2",
     "styled-components": "5.3.0",
-    "twin.macro": "2.4.1"
+    "twin.macro": "2.6.2"
   },
   "devDependencies": {
     "@madebywild/eslint-config": "0.2.2",
     "@madebywild/prettier-config": "0.1.0",
-    "@next/bundle-analyzer": "10.2.0",
+    "@next/bundle-analyzer": "11.0.1",
     "@svgr/webpack": "5.5.0",
-    "@tailwindcss/aspect-ratio": "0.2.0",
-    "@types/node": "15.3.0",
-    "@types/react": "17.0.6",
-    "@types/styled-components": "5.1.9",
-    "@types/webpack-env": "1.16.0",
-    "eslint": "7.26.0",
+    "@tailwindcss/aspect-ratio": "0.2.1",
+    "@types/node": "16.4.11",
+    "@types/react": "17.0.15",
+    "@types/styled-components": "5.1.11",
+    "@types/webpack-env": "1.16.2",
     "babel-plugin-twin": "1.0.2",
-    "husky": "6.0.0",
-    "lint-staged": "11.0.0",
-    "prettier": "2.3.0",
+    "eslint": "7.32.0",
+    "husky": "7.0.1",
```

Comment:
In newer versions of `husky` the `.gitignore` inside the `husky` folder is no longer needed.

Reply:
Good catch :+1: I have removed it.